bridge/client/app/_components/ktb-project-tile/ktb-project-tile.component.html

```diff
@@ -1,7 +1,12 @@
 <dt-tile [id]="project.projectName" [routerLink]="['/project', project.projectName]" *ngIf="project">
-  <dt-tile-title [textContent]="project.projectName"></dt-tile-title>
+  <dt-tile-title [textContent]="project.projectName" [class.error]="project.isShipyardNotSupported()"></dt-tile-title>
   <dt-tile-subtitle>
     <p class="mt-0 mb-2" *ngIf="project.stages"><span [textContent]="project.stages.length"></span> Stages, <span [textContent]="project.getServices().length"></span> Services</p>
+    <p class="mt-0 mb-2" *ngIf="project.getShipyardVersion() as sv">Shipyard version: <span [textContent]="sv"></span> <dt-icon name="abort" class="error bottom" *ngIf="project.isShipyardNotSupported()"></dt-icon> </p>
+    <div class="mb-2" fxLayout="row" fxLayoutAlign="flex-start center" *ngIf="project.isShipyardNotSupported()">
+      <dt-icon name="information" class="info mr-1"></dt-icon>
+      <p class="small m-0">Shipyard version not supported. <br/>Please use <a href="https://keptn.sh/docs/0.7.x/operate/upgrade/" target="_blank" rel="noopener noreferrer" (click)="$event.stopPropagation()">keptn upgrade project</a> to upgrade to the latest shipyard version.</p>
```

Comment:
Not 100% sure, but I believe the link here should point to 0.8.x, right @johannes-b ?

Reply:
This will be addressed in https://github.com/keptn/keptn/issues/3205